packages/schematics/src/collection/application/index.ts

```diff
@@ -268,6 +269,14 @@ export default function(schema: Schema): Rule {
         routing: false
       }),
 
+      filter(
```

Comment:
I think having a function within `/packages/schematics/src/utils/rules` would help reuse it easily.